### PR TITLE
fix build, remove __specs__ from package, fix requires due to @types/…

### PR DESCRIPTION
**Pull request by potapovDim**
…node

### CODE REVIEW

File: dist/helpers/fs.helper.d.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { promptObjectInterface } from "./prompt.helper";
```
Comment (Gennadiii) [inline dist/helpers/fs.helper.d.ts]: Why do you need dts file for helper which is not exported?

File: __tests__/src/helpers/fs.helper.test.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import {beforeEach, describe, expect, it, jest} from "@jest/globals";
-import fs from "fs";
-import {fsHelper} from "../fs.helper";
-import {promptObjectInterface} from "../prompt.helper";
+import * as fs from "fs";
```
Comment (Gennadiii) [inline __tests__/src/helpers/fs.helper.test.ts:2]: You don't need that * as stuff

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tests-selector",
-  "version": "1.0.0",
+  "version": "1.1.3",
```
Comment (Gennadiii) [inline package-lock.json:3]: Please change 1.1.3 -> 1.1.4 so I can publish this version

File: tsconfig.json [hunk 4]
```diff
@@ -1,30 +1,20 @@
 {
-  "compileOnSave": true,
-  "declaration": true,
-  "types": "index.d.ts",
   "compilerOptions": {
-    "incremental": true,
```
Comment (Gennadiii) [inline tsconfig.json:6]: Why did you remove all of that? I kinda liked it )

**Review by Gennadiii — CHANGES_REQUESTED** [review]

File: __tests__/src/helpers/fs.helper.test.ts (hunk 2 above)
Comment (potapovDim) [reply __tests__/src/helpers/fs.helper.test.ts:2]: is is required when import as a namespace

File: dist/helpers/fs.helper.d.ts (hunk 1 above)
Comment (potapovDim) [reply dist/helpers/fs.helper.d.ts]: i think, it can be use in case of direct requirement

File: package-lock.json (hunk 3 above)
Comment (potapovDim) [reply package-lock.json:3]: in package-lock ? maybe it would be better to remove package-lock ?

File: tsconfig.json (hunk 4 above)
Comment (potapovDim) [reply tsconfig.json:6]: is this required thing for something ?

File: package-lock.json (hunk 3 above)
Comment (Gennadiii) [reply package-lock.json:3]: My bad. I thought it was package. But package version still has to be upgraded

File: tsconfig.json (hunk 4 above)
Comment (Gennadiii) [reply tsconfig.json:6]: incremental option with tsBuildInfoFile builds faster, target 2015 is to support older js versions, esModuleInterop allows to import without "* as". And why do you exclude __tests__ ?

Comment (potapovDim) [reply tsconfig.json:6]: incremental + 
tests is not required for package publishing (less library package weight), and you execute them with ts-node

Comment (potapovDim) [reply tsconfig.json:6]: incremental was there, i had changed from top to bottom position in compilerOptions object(

Comment (Gennadiii) [reply tsconfig.json:6]: I got what you mean. Tests don't need tsconfig because they are executed with ts-node and they don't have to be built. Agree

**Review by potapovDim — COMMENTED** [review]
fixed

File: package-lock.json (hunk 3 above)
Comment (potapovDim) [reply package-lock.json:3]: @Gennadiii i think you will do it before publishing ?

Comment (Gennadiii) [reply package-lock.json:3]: Yeah I will
